ab2d-fhir/src/test/java/gov/cms/ab2d/fhir/PatientIdentifierUtilsTest.java

```diff
@@ -177,6 +178,225 @@ void testGetCurrentMbiWrongType() {
         assertNull(IdentifierUtils.getCurrentMbi(List.of(patientIdentifier)));
     }
 
+    @Test
+    void testGetCurrentMbiTypeNotExists() {
+        PatientIdentifier patientIdentifier = new PatientIdentifier();
+        patientIdentifier.setType(null);
+        patientIdentifier.setValue("test-1");
+        assertNull(IdentifierUtils.getCurrentMbi(List.of(patientIdentifier)));
+    }
+
+    @Test
+    void testGetCurrencyFromTypeCodingExtensionReturnsCurrent() {
+        Patient patient = new Patient();
```

Comment:
You can extract common initialization and setup to private method

Reply:
Could you elaborate a bit more here? I think I understand. I should create a different method that will handle the creation of patient for all of these? Then I can return patient from that?

Reply:
No worries. I think I might have addressed this in some setup methods for Coding, Extension, Identifier and Patient.